sbin/pfmon

```diff
@@ -382,6 +384,24 @@ sub normal_sighandler {
     $running = 0;
 }
 
+=head2 reload_config
+
+=cut
+
+sub reload_config {
+    if ( pf::cluster::is_management ) {
+        $process = $TRUE;
```

Comment:
I'm not really sure this will change it in the child processes, meaning we will have to send the signal to all pfmon processes. @jrouzierinverse can probably assert this much better than me.